examples/oidc/README.md

```diff
@@ -27,6 +31,11 @@ Here is an architecture diagram showing how Single Log Out works for this exampl
 
 ![](oidc-slo.png)
 
+### IMAP/SMTP
+Here is an architecture diagram showing how to authenticate JAMES IMAP/SMTP using OIDC Provider:
+
+![](IMAP-SMTP-oidc.png)
```

Comment:
Please also include the source file for the images you generated. This ease changes on those images.

Reply:
Yes, I did add it.